### ISSUE TITLE
Shift brings keyboard focus back after it is ended

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#367]
**Test device**
Dell
**Operating System**
Win
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/617.
When you have keyboard focus on both hands, remove focus, and then hit just the shift key, the both hands focus comes back. Tab was not used, so keyboard focus should stay off in this case.
**Steps to reproduce**
1. Use tab to highlight both hands
2. Use mouse to drag a hand and end keyboard focus
3. Hit the shift key

**Visuals**

https://user-images.githubusercontent.com/41024075/110521567-1f7f0080-80cd-11eb-876b-e1e9ae53f233.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.3/phet/ratio-and-proportion_all_phet.html
Version: 1.0.0-rc.3 2021-03-03 03:29:06 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.82 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
Not seeing this in Firefox on the same device.

**Comment by zepumph** [comment]
@jessegreenberg and I designed the focus display system in this exact way. It is a biproduct of decision that had to be made to support iOS + VO description and interaction. I'm going to close this issue, but I would hope that @BLFiedler @terracoda @jessegreenberg would reopen if they feel differently.
